bitnami/kafka/Chart.yaml

```
@@ -42,4 +42,4 @@ maintainers:
 name: kafka
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/kafka
-version: 25.1.12
+version: 25.1.13
```

As this is a new feature, we should bump a minor to correctly follow [semver](https://semver.org/)

```suggestion
version: 25.2.0
```